build/testing/integration.go

```diff
@@ -529,3 +530,32 @@ func azblob(ctx context.Context, client *dagger.Client, base, flipt *dagger.Cont
 
 	return suite(ctx, "readonly", base, flipt.WithExec(nil), conf)
 }
+
+// gcs simulates the Google Cloud Storage service
+func gcs(ctx context.Context, client *dagger.Client, base, flipt *dagger.Container, conf testConfig) func() error {
+	gcs := client.Container().
+		From("fsouza/fake-gcs-server").
+		WithExposedPort(4443).
+		WithExec([]string{"-scheme", "http", "-public-host", "gcs:4443"}).
+		AsService()
+
+	_, err := base.
+		WithServiceBinding("gcs", gcs).
+		WithEnvVariable("STORAGE_EMULATOR_HOST", "gcs:4443").
+		WithExec([]string{"go", "run", "./build/internal/cmd/gcs/...", "-testdata-dir", testdataDir}).
+		Sync(ctx)
+	if err != nil {
+		return func() error { return err }
+	}
+
+	flipt = flipt.
+		WithServiceBinding("gcs", gcs).
+		WithEnvVariable("FLIPT_LOG_LEVEL", "DEBUG").
+		WithEnvVariable("FLIPT_STORAGE_TYPE", "object").
+		WithEnvVariable("FLIPT_STORAGE_OBJECT_TYPE", "gs").
```

Comment:
```suggestion
		WithEnvVariable("FLIPT_STORAGE_OBJECT_TYPE", "googlecloud").
```

I think we should make it more explicit like this